### DISCUSSION TITLE
V23Beta，Wayland模式下窗口不能沿对角线调整大小

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4411] [Troubleshooting | 故障排查]
截图不能截出这样的光标，用相机拍的。
自己发现的问题，系统是昨天刚刚发布的V23Beta版。
Wayland桌面下，我不能沿着对角线调整大小，必须离远点才能调整。
这个问题在X11桌面和20.9版本没有发生。
![IMG_20230517_155116](https://github.com/linuxdeepin/developer-center/assets/130546027/c8788d4d-4cfe-4ddc-9731-7619ea3b6d9e)
![IMG_20230517_155139](https://github.com/linuxdeepin/developer-center/assets/130546027/69337f2d-dfda-4971-9485-32005794e3c1)

**Comment by Decodetalkers** [comment]
@justforlxz cc
